portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -33,27 +35,88 @@
    *part of the JSON response.
   */
   private String welcome = "Welcome to my portfolio!";
-  private ArrayList<String[]> data;
+
+  // An array list containing all the data that will be recieved with a Get request.
+  private ArrayList<Object[]> data;
+  
+  /**
+   * A linked list that will store every comment on the site.
+   *
+   * A linked list is used instead of an ArrayList in order to ensure that a deletion 
+   * implementation occurs faster in cases where the comment to be deleted is in the
+   * middle of the list.
+   */
+  private LinkedList<Comment> comments = new LinkedList<Comment>();;
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    //Initialize Linked List/s
-    data = new ArrayList<String[]>();
+    // Initialize Linked List/s
+    data = new ArrayList<Object[]>();
     buildIntroData();
+    buildCommentData();
+
+    // Get comment data from the Datastore
+    Query query = new Query("cmt");
```

Comment:
I wouldn't change this now (since you already have comments persisted), but next time I would refrain from using shorthand to describe the entity (instead use something like "comment"). In general I think it's better to be verbose to increase the legibility of your project for others